Avoid stack scanning: unportable and undefined behavior, frequently breaks
with compiler updates.

Index: libschily/shlschily.mk
--- libschily/shlschily.mk.orig
+++ libschily/shlschily.mk
@@ -11,7 +11,7 @@ VPATH=		.:stdio:$(ARCHDIR)
 INSDIR=		lib
 TARGETLIB=	schily
 CPPOPTS +=	-Istdio
-CPPOPTS +=	-DUSE_SCANSTACK	# Try to scan stack frames
+CPPOPTS +=	-DNO_SCANSTACK	# Try to scan stack frames
 CPPOPTS +=	-DPORT_ONLY	# Add missing funcs line snprintf for porting
 include		Targets
 LIBS=		-lc
